### PR TITLE
Updating welcome center title

### PR DESCRIPTION
**Pull request by ettaboyle**
Tuvy caught this and it needed to be updated

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ GitHub Community is built to support all GitHub users on their educational journ
 
 ### Quick Start
```
Comment (Arekn0870) [inline profile/README.md:6]: 5

File: profile/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ GitHub Community is built to support all GitHub users on their educational journ
...
 
-* [Welcome Center](https://github.com/orgs/github-community/discussions)
```
Comment (Arekn0870) [inline profile/README.md:8]: 55

**Review by Arekn0870 — COMMENTED** [review]
4

File: profile/README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ GitHub Community is built to support all GitHub users on their educational journ
 
 ### Quick Start
```
[the remark this answers is not in the document]
Comment (Fart10) [reply profile/README.md:7]: 👌

**Comment by Fart10** [comment]
👌

**Review by Fart10 — COMMENTED** [review]
👌

File: profile/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (pollyapp95-jpg) [reply profile/README.md:7]: pollyapp95@gmail.com

**Pull request merged**